### PR TITLE
[Merged by Bors] - feat: add cdk publish

### PR DESCRIPTION
**Pull request by digikata**
cdk publish
fluvio hub connector list

### CODE REVIEW

**Comment by digikata** [comment]
Need to add package visibility field to Connector config

File: crates/fluvio-hub-protocol/src/constants.rs [hunk 1]
```diff
@@ -4,11 +4,17 @@ pub const CLI_CONFIG_HUB: &str = "hub";
...
 pub const HUB_API_ACT: &str = concatcp!(HUB_API_V, "/action");
 pub const HUB_API_HUBID: &str = concatcp!(HUB_API_V, "/hubid");
+
+// sm specific api
+pub const HUB_API_SM: &str = concatcp!(HUB_API_V, "/pkg/pub");
 pub const HUB_API_LIST: &str = concatcp!(HUB_API_V, "/list");
 
+// connector specific api
+pub const HUB_API_CONN_PKG: &str = concatcp!(HUB_API_V, "/connector/pkg");
+pub const HUB_API_CONN_LIST: &str = concatcp!(HUB_API_V, "/connector/pkg");
```
Comment (galibey) [inline crates/fluvio-hub-protocol/src/constants.rs:16]: "/connector/pkg" -> "/connector/list"

Comment (digikata) [reply crates/fluvio-hub-protocol/src/constants.rs]: thanks, fixed in latest push

**Comment by digikata** [comment]
Merge for internal use - sys arch support to come in a separate pr

**Review by galibey — APPROVED** [review]
LGTM

**Comment by digikata** [comment]
rebase and clippy fix

**Pull request closed without merging**